docs/product-user-guide/src/main/asciidoc/decision-tables-rule-set-entries-ref.adoc

```diff
@@ -0,0 +1,56 @@
+[id='decision-tables-rule-set-entries-ref']
+= RuleSet definitions
+
+Entries in the `RuleSet` area of a decision table define DRL constructs and rule attributes that you want to apply to all rules in a package (not only in the spreadsheet). Entries must be in a vertically stacked sequence of cell pairs, where the first cell contains a label and the cell to the right contains the value. A decision table spreadsheet can have only one `RuleSet` area.
+
+The following table lists the supported labels and values for `RuleSet` definitions.
+
+*<@Michael (4 of 5): Back to the "Initial" question. For the other two tables, the attributes and the `RuleTable` definitions, the user can use a custom label so long as it begins with the letter listed in the table (weird as that is). Are there not also supported "Initials" for the `RuleSet` table (below)? Odd that we provide it in the other tables but not this one.>*
```

Comment:
@sterobin No, only the attributes that can be used in RuleTable have "Initial" support (weird as that is too!)